File: coco/src/main/java/coco/ep/m7i2/Mult.java
package coco.ep.m7i2;

public interface Mult<FT> extends coco.ep.m7.Mult<FT>, coco.ep.m3.Mult<FT>, coco.ep.m7i2.Exp<FT> {

    default coco.ep.Exp<FT> multby(coco.ep.Exp<FT> other) {
        return this.mult(this, convert(other));
    }
    default coco.ep.Exp<FT> powby(coco.ep.Exp<FT> other) {
        return this.power(this, convert(other));
    }

}

File: coco_csharp/CoCo/EP/M7I2/Mult.cs
namespace EP.M7I2 {
    public interface Mult<FT> : Exp<FT>, EP.M7.Mult<FT> {
        EP.Exp<FT> EP.I1.Exp<FT>.multby(EP.Exp<FT> other) {
            return mult(this, convert(other));
        }
        EP.Exp<FT> EP.M7.Exp<FT>.powby(EP.Exp<FT> other) {
            return power(this, convert(other));
        }
    }
}